using-mobile-extensions/mobile-core/identity/README.md

```diff
@@ -217,7 +217,7 @@ NSURL* url = [[NSURL alloc] initWithString:@"www.myUrl.com"];
 }];
 ```
 
-Alternately, starting with SDK version 2.3.0, you can call [getUrlVariables](./identity-api-reference#geturlvariables-ios) and build your own URL:
+Alternately, starting with SDK version 2.3.0 (ACPIdentity version 2.1.0), you can call [getUrlVariables](./identity-api-reference#geturlvariables-ios) and build your own URL:
```

Comment:
Why is this text italicized?

Reply:
Interesting. Typora uses "objective-c" for syntax highlighting in code blocks, but it messes with the parsing in GitHub's diff.  Looks like other places in our docs use "objectivec" instead, so I changed it to that.